### PR TITLE
Doc: update RaidbossGuide.md

### PR DESCRIPTION
**Pull request by MaikoTan**

### CODE REVIEW

File: docs/RaidbossGuide.md [hunk 1]
```diff
@@ -446,6 +454,46 @@ This deprecation applies only to work intended for the cactbot repository.)
 Use of canned conditions and responses is recommended where possible, although
 given Square's extremely talented fight design team, it's not always going to *be* possible.
 
+## Outputs
+
+In order to reduce duplications across trigger sets,
+cactbot built a set of locale strings including some text wisely used between triggers.
```
Comment (quisquous) [inline docs/RaidbossGuide.md:460]:
```suggestion
cactbot has a set of locale strings that includes text repeatedly used by triggers.
```

File: docs/RaidbossGuide.md [hunk 2]
```diff
@@ -446,6 +454,46 @@ This deprecation applies only to work intended for the cactbot repository.)
...
+So we can easily reference to those translated text instead of translating/copy it one more time.
+
+A simple example using `outputStrings` and `Outputs` as below:
```
Comment (quisquous) [inline docs/RaidbossGuide.md:463]: I'm a little skeptical that this is a simple example, considering it requires a helper function that isn't listed in the example.

What about using `E9S Zero-Form Devouring Dark` as an example that combines Outputs and a custom output string, but without logic that's very complicated?

File: docs/RaidbossGuide.md [hunk 3]
```diff
@@ -61,6 +63,11 @@ These files live alongside their parent trigger file in the appropriate folder.
 
 **timeline**
 Optional extra lines to include as part of the timeline.
+The value may be string or an array of strings,
```
Comment (quisquous) [inline docs/RaidbossGuide.md:66]:
```suggestion
The value may be a string or an array of strings,
```

File: docs/RaidbossGuide.md [hunk 4]
```diff
@@ -61,6 +63,11 @@ These files live alongside their parent trigger file in the appropriate folder.
...
+or a `function(data)` that returns string or an array of strings,
+or an array contains different kinds of items above.
+
+There is is a complete example uses **timeline** property in [test.js](../ui/raidboss/data/00-misc/test.js).
```
Comment (quisquous) [inline docs/RaidbossGuide.md:70]:
```suggestion
There is a complete example that uses the **timeline** property in [test.js](../ui/raidboss/data/00-misc/test.js).
```

File: docs/RaidbossGuide.md [hunk 5]
```diff
@@ -446,6 +454,46 @@ This deprecation applies only to work intended for the cactbot repository.)
 Use of canned conditions and responses is recommended where possible, although
 given Square's extremely talented fight design team, it's not always going to *be* possible.
 
+## Outputs
+
+In order to reduce duplications across trigger sets,
+cactbot built a set of locale strings including some text wisely used between triggers.
+So we can easily reference to those translated text instead of translating/copy it one more time.
```
Comment (quisquous) [inline docs/RaidbossGuide.md:461]:
```suggestion
When writing triggers, prefer using `Outputs` if possible to avoid duplication.
```

**Review by quisquous — COMMENTED** [review]
Thank you for the documentation update! I have a couple of small grammar tweaks, and one suggestion about the example:

File: docs/RaidbossGuide.md (hunk 2 above)
Comment (MaikoTan) [reply docs/RaidbossGuide.md:463]: Sure!

**Pull request merged**